### ISSUE TITLE
PHP 8.2 support

### ISSUE THREAD

**Issue by vbezruchkin** [cyrildewit/eloquent-viewable#284]
Please add PHP 8.2 support:

```
PHP Deprecated:  Using ${var} in strings is deprecated, use {$var} instead in 
./vendor/cyrildewit/eloquent-viewable/src/InteractsWithViews.php on line 74

Deprecated: Using ${var} in strings is deprecated, use {$var} instead in 
./vendor/cyrildewit/eloquent-viewable/src/InteractsWithViews.php on line 74
```

Thanks!

**Comment by cyrildewit** [comment]
Hey,

This is fixed by #280 right? The master branch still needs to be made stable. Maybe you can work on that part.

See my comment https://github.com/cyrildewit/eloquent-viewable/issues/283#issuecomment-1476837501 for more context.

Thanks.

**Comment by cyrildewit** [comment]
I will close this in favor of #283
